templates/static/landing.html

```diff
@@ -40,7 +40,7 @@
 
       <div class="row mb-5">
         <div class="col-md-12">
-          <h1><img src="http://assets.freenode.net/freenode/logos/coloured-wblur-alphabg.svg"></h1>
+          <h1><img src="https://assets.freenode.net/freenode/logos/coloured-wblur-alphabg.svg"></h1>
```

Comment:
Shall I go ahead and host these images on `jobs.freenode.net` itself? Seems sub-optimal to link them cross domain :)